stdlib/public/SwiftShims/swift/shims/CMakeLists.txt

```diff
@@ -70,15 +70,12 @@ add_custom_target("copy_shim_headers" ALL
     DEPENDS "${outputs}"
     COMMENT "Copying SwiftShims module to ${output_dir}")
 
-if ("${LLVM_PACKAGE_VERSION}" STREQUAL "")
+if (NOT CLANG_VERSION_MAJOR)
   message(FATAL_ERROR
```

Comment:
This caused the Windows build to fail

```
CMake Error at C:/Users/swift-ci/jenkins/workspace/swift-PR-windows/swift/stdlib/public/SwiftShims/swift/shims/CMakeLists.txt:74 (message):
  CLANG_VERSION_MAJOR must be set in order to infer the path to clang headers
```

I should likely fallback to `LLVM_VERSION_MAJOR` like Clang is doing (e.g. https://github.com/llvm/llvm-project/blob/main/cmake/Modules/GetClangResourceDir.cmake)

Reply:
Tackled in 99fb4e17bcf34f8a9de3d6cb9f9a238da4aab978

Reply:
Better, but the warning string is off

```
CMake Warning at C:/Users/swift-ci/jenkins/workspace/swift-PR-windows/swift/stdlib/public/SwiftShims/swift/shims/CMakeLists.txt:79 (message):
  CLANG_VERSION_MAJOR is not defined, falling back to

               LLVM_VERSION_MAJOR to inder the path to clang headers
```

Need to use [continuation](https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#quoted-argument) appropriately.